src/NuGet.Core/NuGet.Commands/CommandRunners/PackCommandRunner.cs

```diff
@@ -951,7 +951,7 @@ private IEnumerable<IPackageFile> RemoveDefaultExclusions(ICollection<IPackageFi
                 var fileName = Path.GetFileName(filePath);
 
                 return fileName.EndsWith(".nupkg", StringComparison.OrdinalIgnoreCase)
-                    || (fileName.StartsWith(".") && fileName.IndexOf('.', startIndex: 1) == -1);
+                    || (fileName.StartsWith(".", StringComparison.Ordinal) && fileName.IndexOf('.', startIndex: 1) == -1);
```

Comment:
This looks like a path, but it's just comparing a colon '.'. Let's keep ordinal in both string operations

```suggestion
                    || (fileName.StartsWith(".", StringComparison.Ordinal) && fileName.IndexOf('.', startIndex: 1, StringComparison.Ordinal) == -1);
```